app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{Name: " + self.name + ", Health: " + str(self.health) + \
+               ", Hidden: " + str(self.hidden) + "}"
```

Comment:
Use `()` to split lines, not `\`

Reply:
please fix this like Olena says

Example:
```
        return ("Слава Україна!🇺🇦"
                "Героям Слава!🇺🇦")
```

Also use `f` string in this case